pkg/stream/producer_daemon_partitioned_aggregator_test.go

```diff
@@ -33,10 +34,10 @@ func (s *producerDaemonPartitionedAggregatorTestSuite) SetupTest() {
 	var err error
 	next := 0
 	expectedExplicitHashKeys := []string{
-		"42535295865117307932921825928971026431",  // floor((2 ^ 128 - 1) / 4 / 2)
```

Comment:
Shouldn't we keep these comments?

Reply:
ya, I guess..  just needed to adjust the comment so that the linter is not complaining about "commentedOutCode"